### PR TITLE
Fix set user

### PR DESCRIPTION
**Pull request by bandvov**

### CODE REVIEW

File: client/src/components/app-header-nav-left/App-header-nav-left.js [hunk 1]
```diff
@@ -26,35 +35,43 @@ const AppHeaderNavLeft = ({
...
+  const filterRemoveCategoriesHandler = () => {
     filterRemoveAllCategories();
-    setIsShown('')
-  }
+    setIsShown("");
+  };
 
   const filterAddCatalog = (catalog) => () => {
-    filterByName('');
+    filterByName("");
```
Comment (oleksandrhodzevych) [inline client/src/components/app-header-nav-left/App-header-nav-left.js]: "" replace ''

File: client/src/components/app-header-nav-left/App-header-nav-left.js [hunk 2]
```diff
@@ -83,8 +103,10 @@ const AppHeaderNavLeft = ({
     <nav className="nav-bar nav-left">
       <ul>
         {items}
-        <li><Link to="/news" >News</Link></li>
-        </ul>
+        <li>
+          <Link to="/news">News</Link>
```
Comment (oleksandrhodzevych) [inline client/src/components/app-header-nav-left/App-header-nav-left.js]: hard coded

**Review by oleksandrhodzevych — CHANGES_REQUESTED** [review]

File: client/src/components/app-header-nav-left/App-header-nav-left.js [hunk 3]
```diff
@@ -11,10 +11,19 @@ import {
   filterByName,
   setCatalogFilter,
   clearFilter,
-} from '../../actions';
-import withStoreService from '../hoc';
-import AppHeaderNavLeftItem from '../app-header-nav-left-item';
-import AppHeaderNavLeftItemDropDown from '../app-header-nav-left-item-dropdown';
+  setUser,
+} from "../../actions";
```
Comment (oleksandrhodzevych) [inline client/src/components/app-header-nav-left/App-header-nav-left.js]: @/folder1/folder2/actions

**Review by oleksandrhodzevych — CHANGES_REQUESTED** [review]

File: client/src/components/app-header-nav-left/App-header-nav-left.js (hunk 1 above)
Comment (bandvov) [reply client/src/components/app-header-nav-left/App-header-nav-left.js]: fixed

File: client/src/components/app-header-nav-left/App-header-nav-left.js (hunk 2 above)
Comment (bandvov) [reply client/src/components/app-header-nav-left/App-header-nav-left.js]: fixed

**Pull request merged**